### PR TITLE
Feature/conf

### PR DESCRIPTION
**Pull request by jensborch**

### CODE REVIEW

File: core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import java.time.ZonedDateTime;
+import java.time.temporal.TemporalUnit;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Configuration for Webhooks4j. Users of Webhooks4j must implement a CDI
+ * producer that returns implementation of this interface.
+ * <p>
+ * Note: The TTL values are only used by the MongoDB modules, but could be used
```
Comment (thaestrup) [inline core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java:11]: The note about the TTL should be moved to getTimeToLive documentation

File: core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ *
+ */
+public interface WebhookConfiguration {
+
+    /**
+     * Time to live (TTL) for webhook events in the events repository.
+     *
+     * @return the TTL
+     */
+    long getTimeToLive();
```
Comment (thaestrup) [inline core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java:23]: I would call the method getTTL

File: core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    /**
+     * Time unit used for TTL.
+     *
+     * @return the time unit
+     */
+    TimeUnit getTimeToLiveUnit();
+
+    /**
+     * The synchronisation offset to use when synchronizing old events.
```
Comment (thaestrup) [inline core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java:33]: The offset here is used to subtract from the last received event to get a safer starting point. I find the word "offset" a bit confusing in this context. 
I was thinking more in the line of "lookBackTime"

Comment (jensborch) [reply core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java:33]: I think offset can be used as it is is defined as "a amount that diminishes or balances the effect of an opposite one"

**Review by thaestrup — APPROVED** [review]

**Pull request merged**